### ISSUE TITLE
Unable to find a matching configuration of project :react-native-share-pdf

### ISSUE THREAD

**Issue by nilsbacke-ef** [MadeinFrance/react-native-share-pdf#6]
**Describe the bug**
```
Could not determine the dependencies of task ':app:preDebugBuild'.
> Could not resolve all task dependencies for configuration ':app:debugRuntimeClasspath'.
   > Could not resolve project :react-native-share-pdf.
     Required by:
         project :app
      > Unable to find a matching configuration of project :react-native-share-pdf:
          - None of the consumable configurations have attributes.
```

**To Reproduce**

- [ ] I checked the demo project and cannot reproduce the issue
- [ ] I checked the demo project and the issue exists

Neither - the demo project does not run.

1. Create a new project. 
2. Follow the installation instructions for android (including `yarn add react-native-share-pdf`)

`dependencies` in `app/build.gradle`:
```
dependencies {
    implementation fileTree(dir: "libs", include: ["*.jar"])
    implementation "com.android.support:appcompat-v7:${rootProject.ext.supportLibVersion}"
    implementation "com.facebook.react:react-native:+"  // From node_modules
    implementation project(':react-native-share-pdf')
}
```

`settings.gradle`
```
rootProject.name = 'SharePDFTester'

include ':app'
include ':react-native-share-pdf'
project(':react-native-share-pdf').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-share-pdf/android')
```

in `MainApplication.java`
```
@Override
    protected List<ReactPackage> getPackages() {
      return Arrays.<ReactPackage>asList(
          new MainReactPackage(),
          new ShareFilePackage()
      );
    }
```

**Comment by thanhtamha** [comment]
i have this issue today too.

**Comment by deenMuhammad** [comment]
the issue is because the node_modules/react-native-share-pdf/android does not have files build.gradle and project.properties

**Comment by geovannyPaiz** [comment]
where can I found that files?

**Comment by ignaciosarobe** [comment]
I have this same problem too. Any solution?

**Comment by albertolabuto** [comment]
same shit here too

**Comment by masako-yamada** [comment]
I found this fork has all the necessary files: 
https://github.com/longgame/react-native-share-pdf

I'll write here about how I integrated it into my project.

1. Fork the repo to your own github account

2. Edit your `package.json`
"react-native-share-pdf": "github:*_your_account_name_*/react-native-share-pdf",

2. `npm install react-native-share-pdf`

3. Follow the instruction to link the lib

4. Close Android Studio

5. Remove `*.iml` file and `.ideas`  in *your_root_dir*/android

6. Launch Android Studio and open *your_root_dir*/android folder
